src/check/parse/tokenize.zig

```diff
@@ -1280,19 +1280,20 @@ pub fn checkTokenizerInvariants(gpa: std.mem.Allocator, input: []const u8, debug
     tokenizer.tokenize();
     var output = tokenizer.finish_and_deinit();
     defer output.tokens.deinit();
+    _ = debug;
 
-    if (debug) {
-        std.debug.print("Original:\n==========\n{s}\n==========\n\n", .{input});
-    }
+    // if (debug) {
```

Comment:
Rather than commenting these out, if you want to temporarily disable this can you pass debug=false from these tests?

Reply:
Done